### PR TITLE
Include Name Pronuncation Audio From Media Webservice

### PR DESCRIPTION
**Pull request by andrewMETALAB**
This PR makes an update to the profile-banner section (of each faculty member's profile-page) to include name-pronunciation-audio from the media webservice.

To test:

- look at Steve's profile page to see the newly included audio icon next to his name. Click the icon to hear Steve's name. Play around with it and make sure there are no errors or issues.
- look at Deborah Chen's profile page (or most other faculty members) - she has not uploaded an audio file for her name, so there is no audio icon next to her name. Make sure there are no errors or issues.

### CODE REVIEW

File: resources/views/pages/profile/show.blade.php [hunk 1]
```diff
@@ -120,6 +135,53 @@
...
+            function handlePlayButton() {
+                if (audioElem.paused) {
+                    playAudio();
+                } else {
+                    audioElem.pause();
+                }
+            }
+            async function playAudio() {
+                try {
+                    await audioElem.play();
```
Comment (luisjg) [inline resources/views/pages/profile/show.blade.php:165]: Could this be simplified by tacking on .catch() after .play()?

`audioElem.play().catch(() => //something on error)`?

**Review by luisjg — CHANGES_REQUESTED** [review]
I can't point to what the issue is, but the speaker icon on every breakpoint doesn't look right. See attached

Screen Shot 2019-03-29 at 12 07 50 PM

File: resources/views/pages/profile/show.blade.php (hunk 1 above)
Comment (andrewMETALAB) [reply resources/views/pages/profile/show.blade.php:165]: no, unfortunately that produces a syntax error: "missing catch or finally after try"

**Review by luisjg — APPROVED** [review]
👍

**Pull request merged**